fedimint-core/src/task.rs

```diff
@@ -231,49 +222,15 @@ impl TaskGroup {
         };
         let handle = self.make_handle();
 
-        if let Some(handle) = self::imp::spawn_local(name.as_str(), async move {
+        let handle = runtime::spawn_local(name.as_str(), async move {
```

Comment:
Why is `task::spawn_local` still not available in WASM even though the underlying impl supports it now?